ansible_base/lib/dynamic_config/dynamic_settings.py

```diff
@@ -133,6 +133,9 @@
 
     # Permissions a user will get when creating a new item
     ANSIBLE_BASE_CREATOR_DEFAULTS = ['add', 'change', 'delete', 'view']
+    # Permissions API will check for related items, think PATCH/PUT
+    # This is a precedence order, so first action related model has will be used
+    ANSIBLE_BASE_CHECK_RELATED_PERMISSIONS = ['use', 'change', 'view']
```

Comment:
Is this an AND or OR combination, meaning does the user need at least one of these permissions or all of them to be able to use the object as related?

Reply:
Formally I think this is neither AND or OR. It is first-found. Because there are 2 conditions here:
 - Is there a permission for this action, for this model
 - Given the prior criteria is satisfied, does the user have that permission

That's why it is a precedence order. First it determines if the model has a "use" permission, and if it does, the answer is a True/False the user has "use" permission. But for _models_ that don't have that first option, it will go on to "change" and if the model has the permission, return True/False user has "change" permission, and so-on.